src/graph_compute/Makefile

```diff
@@ -1,7 +1,7 @@
 CC  ?= gcc
 CXX ?= g++
 CFLAGS = -O3 -Wall 
-CXXFLAGS = -fcilkplus -std=c++11 -O3 -Wall -m64 -march=native -mtune=native -pthread -ffast-math -fgcse-las
+CXXFLAGS = -fcilkplus -std=c++11 -O3 -Wall -m64 -march=native -mtune=native -pthread -ffast-math -fgcse-las -msse4.2
```

Comment:
Is this not enabled by default by `-march=native`? Also, can we use SSE4.2 anywhere?